src/main/frontend/src/utils/itemService.js

```diff
@@ -1,11 +1,45 @@
+import axios from "axios";
 const host = "localhost:8080";
 
 export const itemsFetch = async (page, size) => {
-  const items = await fetch(`http://${host}/api/v1/items?page=${page}&size=${size}`)
+  const items = await fetch(
+    `http://${host}/api/v1/items?page=${page}&size=${size}`
+  );
   return items.json();
 };
 
 export const itemsFetchByDate = async (page, size, order, orderColumn) => {
-  const items = await fetch(`http://${host}/api/v1/items?page=${page}&size=${size}&order=${order}&orderColumn=${orderColumn}`)
+  const items = await fetch(
+    `http://${host}/api/v1/items?page=${page}&size=${size}&order=${order}&orderColumn=${orderColumn}`
+  );
   return items.json();
 };
+
+export const itemsFetchById = async (id) => {
+  return axios
+    .get(`http://${host}/api/v1/items/itemcic?itemId=${id}`)
+    .then((res) => {
+      return res.data;
+    })
+    .catch((error) => {
+      return error.response;
+    });
+};
+
+export const itemBid = async (token, item) => {
```

Comment:
Pay attention to this method, in spring security you allowed all requests with /api/v1/items to be allowed to unauthenticated users.